terraform/templates/prometheus.yml

```diff
@@ -13,8 +13,13 @@ alerting:
 
 # Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
 rule_files:
-  - "rules/blockchain_alerts.yml"
   - "rules/recording_rules.yml"
+  %{ if workspace == "dev" }
```

Comment:
Why does this need to be specific to dev?

Reply:
The idea was to separate dev and prod alerts, but perhaps we can have prod alert on both. I was under the impression that prod might not have the same level of traffic so we wouldn't need to alert on low tx rate but rather reuse the consensus block metric. Maybe @sherry-x can chime in?

Reply:
I think we can start with using same alert config, if it's too noisy then we create separate one. you just need to enable the alert on dev for oncall. Like you mentioned, it's a pagerduty config to enable this.

Reply:
Thanks for the comments @sherry-x @mgorven -- I've gone ahead and updated.